hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/BalanceOfPrecompile.java

```diff
@@ -50,6 +51,8 @@ public TransactionBody.Builder body(
 
     @Override
     public Bytes getSuccessResultFor(final ExpirableTxnRecord.Builder childRecord) {
+        Objects.requireNonNull(balanceWrapper);
```

Comment:
For all of these `requireNonNull` calls I think we should use the variant with a message alluding to why it could be null...
```suggestion
        Objects.requireNonNull(balanceWrapper, "`body` method should be called before `getSuccessResultsFor`");
```